README.md

```diff
@@ -143,105 +143,16 @@ The following commands are supported for profiles:
-| **SAS Path** | Path to SAS Executable on the server | Must be a fully qualified path on the SSH server to a SAS executable |
-
-#### Add New SAS Profile
-
-Open the command palette (`F1`, or `Ctrl+Shift+P` on Windows or Linux, or `Shift+CMD+P` on OSX). After executing the `SAS.addProfile` command, select a connection type and complete the prompts to create a new profile. For SAS Viya Connections, depending on your SAS version, the values for the prompts differ slightly.
-
-- For SAS Viya 2022.11 and later, you can leave Client ID and Client Secret prompts empty and simply press Enter. (The built-in Client ID `vscode` will be used.)
-- For SAS Viya 2022.10 and before (including SAS Viya 3.5), you need to provide a Client ID and secret.
-
-For more information about Client IDs and the authentication process, please see the blog post [Authentication to SAS Viya: a couple of approaches](https://blogs.sas.com/content/sgf/2021/09/24/authentication-to-sas-viya/). A SAS administrator can follow the Steps 1 and 2 in the post to register a new client.
-
-#### Delete Connection Profile
-
-After executing the `SAS.deleteProfile` command:
-
-1. Select profile to delete from the list of profiles
-2. A notification message will pop up on successful deletion
-
-#### Switch Current Connection Profile
-
-After executing the `SAS.switchProfile` command:
-
-1. If no profiles can be found, the extension will ask to [create a new profile](#add-new-sas-profile)
-2. Select profile to set active from the list of profiles
-3. The StatusBar Item will update to display the name of the selected profile
-
-#### Update Connection Profile
-
-Update profile gives the ability to modify existing profiles, including updating from password to token flow and vice versa.
-
-After executing the `SAS.updateProfile` command:
-
-1. Select profile to update from the list of profiles
-2. Complete the prompts to update profile
-
-To update the name of a profile, please delete and recreate it.
-
-#### Notes:
-
-- There is a potential issue with switching between multiple profiles on Windows. For more information, see [#215](https://github.com/sassoftware/vscode-sas-extension/issues/215).
 
 ### Running SAS Code
 
 After configuring the SAS extension for your SAS environment, run your SAS program and view the log and results.
 
-To run a SAS program:
-
-1. Click the running man icon in the upper right corner of your SAS program window.
-2. For a secure connection to SAS Viya we use an authorization code for authentication. Complete these steps to connect.
-
-   1.1. VS Code may prompt you that the extension wants to sign in. Click 'Allow'.
-
-   1.2. VS Code may prompt you about opening an external web site. Click 'Open'.
-
-   1.3. This will take you to a SAS Logon prompt. Log in with your SAS credentials.
-
-   1.4. SAS returns an authorization code. Copy this code.
+Details on creating and managing profiles is available on the [Connect and Run page](connect-and-run.md).
```

Comment:
perhaps should say Details on running code here

Reply:
Fixed to read 'Details on running code is available......'